Adapter_oM/enums/PushType.cs

```diff
@@ -29,23 +29,22 @@
 
 namespace BH.oM.Adapter
 {
-    [Description("FullCRUD - Calls all CRUD methods as appropriate.\n" +
-        "CreateOnly - Uses only the Create CRUD method to export the objects. This may create duplicates if the object already exists.\n" +
-        "UpdateOnly - Uses only the Update CRUD method to update the objects in the external software. All other objects in the model are left untouched.\n" +
-        "DeleteThenCreate - For all objects being Pushed, identifies their type, calls Delete to remove all of those types, then it Creates them.\n" +
-        "AdapterDefault - Picks the value hard-coded in the Adapter.")]
+    [Description("Controls which of the push methods that should be used by the Adapter.")]
     public enum PushType //add error message if not picked up --> compliance check for this?
     {
         [Description("Calls all CRUD methods as appropriate.")]
         FullCRUD,
         [Description("Uses only the Create CRUD method to export the objects. This may create duplicates if the object already exists.")]
         CreateOnly,
+        [Description("Same as FullCRUD, but does not update pre-existing objects.")]
+        CreateNonExisting,
         [Description("Uses only the Update CRUD method to update the objects in the external software.")]
         UpdateOnly,
         [Description("For all objects being Pushed, identifies their type, calls Delete to remove all of those types, then it Creates them.")]
         DeleteThenCreate,
         [Description("If this is chosen, then the m_AdapterSettings.DefaultPushType is picked.")]
```

Comment:
I had in mind to expose to the user only to the most simple information possible, then leaving a more explicative comment for developers:
```suggestion
        [Description("AdapterDefault - Picks the value hard-coded in the specific Adapter.")] // If this is chosen, then the m_AdapterSettings.DefaultPushType is picked
```